skimage/color/colorconv.py

```diff
@@ -56,7 +56,9 @@
 from warnings import warn
 import numpy as np
 from scipy import linalg
-from ..util import dtype, dtype_limits
+from ..util import dtype, dtype_limits, img_as_ubyte
+from matplotlib import pyplot as plt
```

Comment:
The standard form `import matplotlib.pyplot as plt` would be my preference. Also in prior file.